devtools/conda-envs/test_env.yaml

```diff
@@ -7,30 +7,30 @@ dependencies:
     # Base depends
   - python
   - pip
-
-    # Testing
+  - packaging
+  - openmm
+  - openff-forcefields
+  - smirnoff99Frosst
+  - numpy
+  - networkx
+  - ambertools
+  - rdkit
+  - mdtraj
+  - xmltodict
+    # Test-only/optional/dev
   - pytest
   - pytest-cov
   - nbval
   - codecov
   - coverage
-  - numpy
-  - networkx
-  - ambertools
-  - rdkit
   - openeye-toolkits
-  - packaging
-    # Removed until a new release which targets openff-toolkit is made.
-#  - openmmforcefields
-  - openmm
-  - openff-forcefields
-  - smirnoff99Frosst
+  - black
+  - isort
+  - openmmforcefields
```

Comment:
`openmmforcefields` requires `openff-toolkit`, so this pulls in a stable `openff-toolkit` package. But the tests on `master` require new code since the last release, so the unit tests break. 

So we COULD `conda remove --force openff-toolkit-base`, but we don't even need to do _that_, since none of our unit tests require `openmmforcefields`. Only the example tests do., and they [do run the `conda remove...` line](https://github.com/openforcefield/openff-toolkit/blob/1ba11ebbafa67f7953b13f6c0af6183a5801f33e/.github/workflows/examples.yml#L99) to break the dependency loop. 
```suggestion
```